github-runner-manager/src/github_runner_manager/manager/runner_manager.py

```diff
@@ -522,55 +521,12 @@ def _create_runner(args: _CreateRunnerArgs) -> InstanceID:
                 runner_identity=runner_identity,
                 runner_context=runner_context,
             )
-
-            # This wait should be deleted to make the runner creation as
-            # quick as possible. The waiting should only be done in the
-            # reactive case, before checking that a job was taken.
-            RunnerManager.wait_for_runner_online(
```

Comment:
do you think we should adapt the waiting interval in the reactive consumer ? because we are now reducing the amount of time to wait for a job to be picked up implicitly